CMakeLists.txt

```diff
@@ -233,6 +239,7 @@ endif()
 if (ISLE_USE_DX5)
   target_include_directories(lego1 PRIVATE "${CMAKE_SOURCE_DIR}/3rdparty/dx5/inc")
   target_link_directories(lego1 PRIVATE "${CMAKE_SOURCE_DIR}/3rdparty/dx5/lib")
+  target_link_libraries(lego1 PRIVATE d3drm)
```

Comment:
Mingw needs this as well, so add it to the next `target_link_libraries` unconditionally.

Reply:
Done.